README.md

```diff
@@ -858,9 +858,8 @@ here's the gist of it:
 
 **DO NOT merge from master to your feature branch; rebase.**
 
-Note that all dcrdata.org community and team members are expected to adhere to
-the code of conduct, described in the [CODE_OF_CONDUCT](docs/CODE_OF_CONDUCT.md)
-file. These guidelines are generally not a challenge for decent humans.
+Note that all dcrdata.org community and team members are expected to follow the Code Contribution Guidelines, described in the [Code Contribution Guidelines](https://github.com/decred/dcrd/blob/master/docs/code_contribution_guidelines.md)
```

Comment:
Sorry, I just realized this points so dcrd's guidelines instead of dcrdata's.

Reply:
dcrdata does not have 'code_contribution_guidelines.md' file.
Should I copy it from dcrd repo?
My assumption was that it is the same.

Reply:
It's in the docs folder.

Reply:
that's not the same.
[Code Contribution Guidlines](https://github.com/decred/dcrd/blob/master/docs/code_contribution_guidelines.md) goes more into the details.

Reply:
Indeed, it is not.  It's not for this repo, but we could improve our CONTRIBUTING.md if it seems worthwhile.

Reply:
ok, but that should be another PR because that PR was only supposed to quickly solve the Kubernetes link problem.

Should I create PR for the Contributing page?

Reply:
Only if you feel the contributing page needs improvement.  I'd prefer not to completely overhaul it unless there's good reason to do so.

Reply:
no, I don't think it needs to be rewritten, but can I add this line:

`Further reading: [Code Contribution Guidelines](https://github.com/decred/dcrd/blob/master/docs/code_contribution_guidelines.md) (dcrd repo)`

Reply:
That's a good idea.  Just be sure to link to our contributing.md too.